src/web/utils/textHelpers.spec.ts

```diff
@@ -94,4 +95,17 @@ describe('Text helper tests', () => {
       }
     });
   });
+
+  describe('sortstrings', () => {
```

Comment:
per my comment on the `sortStrings` method - perhaps it's worth covering the case where the strings are numbers to differentiate from natural order:
```
describe('sort number strings alphabetically', () => {
  const testStringArr = [
    ['1', '2', '10'],
    ['1', '10', '2'],
  ];
  it.each(testStringArr)('should return array sorted alphabetically', () => {
    for (const t of testStringArr) {
      expect(sortStrings(t)).toEqual(['1', '10', '2']);
    }
  });
});
```

Reply:
Done